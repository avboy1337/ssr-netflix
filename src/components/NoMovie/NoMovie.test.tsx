import React from 'react';
import { render } from '@testing-library/react';
import NoMovie from './NoMovie';

describe('NoMovie', () => {
    it('render NoMovie module', () => {
        const { getByText } = render(<NoMovie />);
        const noMovieFoundText = getByText(/No Movie Found/i).innerHTML;
        expect(noMovieFoundText).toEqual('No Movie Found');
    });

    it('renders NoMovie snapshot', () => {
        const { asFragment } = render(<NoMovie />);
        // @ts-ignore
        expect(asFragment(<NoMovie />)).toMatchSnapshot();
    });
});
