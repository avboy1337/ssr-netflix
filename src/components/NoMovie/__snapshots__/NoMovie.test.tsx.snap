// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMovie renders NoMovie snapshot 1`] = `
<DocumentFragment>
  <div>
    <p>
      No Movie Found
    </p>
  </div>
</DocumentFragment>
`;
